mozregression/releases.py

```diff
@@ -106,6 +106,31 @@ def date_of_release(release):
         raise UnavailableRelease(release)
 
 
+def tag_of_release(release):
+    """
+    Provide the mercurial tag of a release, suitable for use in place of a hash
+    """
+    if re.match(r'^\d+$', release):
+        release += '.0'
+    if re.match(r'^\d+\.\d(\.\d)?$', release):
+        return 'FIREFOX_%s_RELEASE' % release.replace('.', '_')
+    else:
+        raise UnavailableRelease(release)
+
+
+def tag_of_beta(release):
+    """
+    Provide the mercurial tag of a beta release, suitable for use in place of a
+    hash
+    """
+    if re.match(r'^\d+\.0b\d+$', release):
+        return 'FIREFOX_%s_RELEASE' % release.replace('.', '_')
+    elif re.match(r'^\d+(\.0)?$', release):
+        return 'FIREFOX_RELEASE_%s_BASE' % release.replace('.0', '')
+    else:
+        raise UnavailableRelease(release)
```

Comment:
Just as a point of interest, [buildhub](https://mozilla-services.github.io/buildhub/) would be an alternative option for getting this data, but I think hg tags are a better approach (probably less likely to go down).

Reply:
Ah, thanks for that, I might find it useful elsewhere.